### PR TITLE
*: Add GetMinTS 

### PR DESCRIPTION
**Pull request by rleungx**

### CODE REVIEW

**Comment by iosmanthus** [comment]
Would you add a pull request link for the kvproto update?

File: internal/mockstore/mocktikv/pd.go [hunk 1]
```diff
@@ -122,6 +122,10 @@ func (c *pdClient) GetLocalTSAsync(ctx context.Context, dcLocation string) pd.TS
 	return c.GetTSAsync(ctx)
 }
 
+func (c *pdClient) GetMinTS(ctx context.Context) (int64, int64, error) {
+	return 0, 0, nil
```
Comment (binshi-bing) [inline internal/mockstore/mocktikv/pd.go:126]: Do we need to actually implement it?

Comment (rleungx) [reply internal/mockstore/mocktikv/pd.go:126]: I think we don't need it for now since there is no test to call it.

Comment (binshi-bing) [reply internal/mockstore/mocktikv/pd.go:126]: ok

**Review by binshi-bing — APPROVED** [review]
LGTM

**Comment by rleungx** [comment]
> kvproto

https://github.com/pingcap/kvproto/pull/1112

**Comment by rleungx** [comment]
> Would you add a pull request link for the kvproto update?

https://github.com/pingcap/kvproto/pull/1112

**Comment by rleungx** [comment]
@iosmanthus PTAL

**Review by iosmanthus — APPROVED** [review]

**Pull request merged**